demos/hazard/NonparametricSourcesClassicalPSHA/README.txt

```diff
@@ -0,0 +1,17 @@
+Classical PSHA Non-parametric sources Demo
+
+This calculation demo exercises the Classical hazard caclculator with a reduced
+source model logic tree, containing a single model with a non-parametric sources
+with magnitude larger Mw7.0.
+
+In this example, we first compute hazard curves for all IMTs and sites of
+interest (see below). From each curve set we interpolate hazard maps at 10% and
+2% probabilities of exceedance. Finally, we also extract Uniform Hazard Spectra
+(UHS) for the same probabilities from the PGA and SA hazard maps.
+
+Expected runtime: 
```

Comment:
We should add the expected runtime (what's the reference kind of machine for this kind of estimate?)

Reply:
Your laptop is a good reference machine. Approximate per excess, I would say in this case the runtime is < 20 s